torch/_subclasses/fake_tensor.py

```diff
@@ -444,12 +454,30 @@ def __torch_dispatch__(self, func, types, args=(), kwargs=None):
             else:
                 return args[0].fake_device
 
+        with enable_torch_dispatch_mode(self), torch.overrides.enable_reentrant_dispatch():
+            if func in meta_table:
+                r = meta_table[func](*args, **kwargs)
+                return r
+            elif func in decomposition_table:
+                r = decomposition_table[func](*args, **kwargs)
+                return r
+
+        with no_dispatch():
```

Comment:
Can we gate this on there existing symbolic dims in the tensor until `torch.ops.aten.stride` & co don't give incorrect strides ?

Reply:
should also be done now